README.md

```diff
@@ -20,9 +20,9 @@ A simple widget to collect users' feedback.
 5.  Add widget to your page
 
 ```javascript
-import { createWidget } from '@codexteam/surveys';
+import { CodeXSurveys } from '@codexteam/surveys';
```

Comment:
I think, there should be a default export instead of a named since there are no more exports.

Reply:
Default export is not good, isn't it? https://stackoverflow.com/a/68665805/8894470